tests/embind/embind.test.js

```diff
@@ -378,20 +378,38 @@ module({
     });
 
     BaseFixture.extend("string", function() {
+        var embindStdStringUTF8Support = (Module['EMBIND_STD_STRING_UTF8_SUPPORT'] == true);
```

Comment:
Does this test just one of the code paths? I feel like this diff should include a change to the build system so both compile-time options can be tested.

Reply:
Agree on this, but at the moment I have not the slightest idea how to do this. Currently I keep changing setting.js and re-run the unit tests.

Reply:
Oof, if I recall correctly, @juj integrated the embind tests into the Emscripten suite. IMVU used its own test runner with multiple variants of the tests built. I don't have an Emscripten checkout at the moment, maybe @juj can help?